data/yam/agama/auto/alp_micro.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
for this ticket as discussed offline we will be using a different format than in other previously picked tickets for Agama auto-installation, so you can delete .sh and .json and only provide jsonnet.

Reply:
OK. thanks, will try jsonnet.